### PR TITLE
ramips: routerboard 750gr3 cut unused gpio switch poe_passthrough

### PR DESCRIPTION
**Pull request by john-tho**
This device does not support PoE out,
and GPIO 17 without an additional base does not exist.

```
cat /sys/firmware/mikrotik/hard_config/hw_options
raw		: 0x084a4007
has POE out	: false

logread: export_store: invalid GPIO 17
```
Edit: ~~The DTS label-mac-device alias should no longer be required,
now that MAC addresses are determined by rbsysfs, and assigned in 02_network~~ Commited in 5549b843885fb984bf1ec8c771c4598140eb3b6e 

This was identified while implementing support for 760igs, https://github.com/openwrt/openwrt/pull/3103,
which shares very similar hardware.

Has not been tested on 750gr3.

On the 760igs, the GPIO base is 480.

~~Still WIP~~

### CODE REVIEW

**Comment by adrianschmutzler** [comment]
This touches two completely separate things, and should go into two separate commits.

**Comment by john-tho** [comment]
Will do

**Comment by adrianschmutzler** [comment]
I've picked the label-mac-device change and pushed it to master right away, so you may just reduce this to the gpio stuff.

**Comment by f00b4r0** [comment]
LGTM

**Comment by adrianschmutzler** [comment]
@f00b4r0 Reviewed-by?

**Comment by adrianschmutzler** [comment]
> Has not been tested on 750gr3.

Well, if the _information_ is correct, I don't think the change needs to be _tested_.

So, what's the state? Can this be merged if a Reviewed-by is provided?

**Comment by f00b4r0** [comment]
I don't think a 3-line deletion warrants a reviewed by, but sure, why not.

**Comment by adrianschmutzler** [comment]
The value was introduced here: https://github.com/openwrt/openwrt/commit/6ba58b7b020c4b793e6fb7a43c8adef070579d8d

**Comment by f00b4r0** [comment]
> The value was introduced here: [6ba58b7](https://github.com/openwrt/openwrt/commit/6ba58b7b020c4b793e6fb7a43c8adef070579d8d)

I don't even remember why I'm on the SoB on this patch TBH. I don't own the device, so if the matter is confirming that PoE passthrough is not available (which seems to be the case), then it would probably be best to ping someone who does :)

**Comment by john-tho** [comment]
I have now found discussion of the introduction: https://github.com/openwrt/openwrt/pull/1686#discussion_r244499123
The snippet (now inaccessible full file) has the comment:
`// Used to enable power-over-ethernet passthrough to eth4 (according to OEM source).`
Relevant vendor source here: https://github.com/robimarko/routeros-GPL/blob/b5c44a2d0f939bec0c219ee21e1d549bd6c10cb3/linux_3_3_5_patch_RouterOS644/arch/mips/rb/platform.c#L4026

I added a note to the commit message to try to reflect this.

With this information, I feel this PR is accurate, and can be merged?

Cheers

**Comment by f00b4r0** [comment]
I eventually remembered why I'm on the SoB: I cooked the very initial patch as a PoC to show that this device, which was initially "supported" via an ugly reflash of the bootloader with u-boot, could be natively supported. I was the one who mistakenly added this PoE passthrough support since I didn't own the device and couldn't confirm whether it was useful or not. This was then unfortunately picked up as is by the submitter.

tl;dr: @adrianschmutzler I think this should be merged.

HTH

**Comment by adrianschmutzler** [comment]
Thanks, will be merged soon.
